core/app/models/spree/stock/coordinator.rb

```diff
@@ -61,6 +61,17 @@ def splitters(stock_location)
         # extension point to return custom splitters for a location
         Rails.application.config.spree.stock_splitters
       end
+
+      def stock_locations
+        preferred_stock_locations = @inventory_units.flat_map(&:line_item).uniq.map(&:stock_locations).flatten.uniq
+        preferred_stock_locations.present? ? preferred_stock_locations : StockLocation.active
+      end
+
+      def inventory_units_for_stock_location(stock_location)
+        return inventory_units unless inventory_units.any? {|iu| iu.line_item.line_item_stock_locations.present? }
+        line_item_stock_locations = @unpacked_inventory_units.flat_map{|iu| iu.line_item.line_item_stock_locations.where(stock_location_id: stock_location.id)}.uniq
```

Comment:
Line is too long. [164/100]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.